### ISSUE TITLE
Invalid PW -> misleading error message about LAN/VPN

### ISSUE THREAD

**Issue by cerealcommas** [turnerlabs/samlkeygen#35]
samlkeygen returns the following error message when passed invalid credentials while on LAN/VPN:

```
samlkeygen: Error getting SAML Response. If not on LAN, please VPN in.
```

**Comment by bradodarb** [comment]
> samlkeygen returns the following error message when passed invalid credentials while on LAN/VPN:
> 
> ```
> authen: Error getting SAML Response. If not on LAN, please VPN in.
> ```

I get the same behavior when using the python workflow, but it will authenticate via the dockerized workflow (of course because the $USER param is directly injected).

Workaround is using the positional args for user and wrapping in an alias, but might be nice to have an alternative user env var with fallback for the current $USER

**Comment by cerealcommas** [comment]
> > samlkeygen returns the following error message when passed invalid credentials while on LAN/VPN:
> > ```
> > authen: Error getting SAML Response. If not on LAN, please VPN in.
> > ```
> 
> I get the same behavior when using the python workflow, but it will authenticate via the dockerized workflow (of course because the $USER param is directly injected).
> 
> Workaround is using the positional args for user and wrapping in an alias, but might be nice to have an alternative user env var with fallback for the current $USER

@bradodarb Yep — I'm ultimately able to authenticate, so it's not a blocker. 

But it would probably be good to have separate error messages for "can't authenticate because you're not on LAN and I can't establish a connection" vs "can't authenticate because you passed invalid credentials"

Or even just tweaking the existing error message to remind the user that invalid credentials may be the problem (e.g., "If not on LAN, please VPN in. If you are on LAN, make sure you entered the right username/password.”)
